src/Adapter/Product/QueryHandler/GetProductSupplierOptionsHandler.php

```diff
@@ -62,7 +62,12 @@ public function __construct(GetProductSuppliersForEditingHandler $getProductSupp
     public function handle(GetProductSupplierOptions $query): ProductSupplierOptions
     {
         $product = $this->getProduct($query->getProductId());
-        $productSuppliersForEditing = $this->getProductSuppliersForEditingHandler->handle(new GetProductSuppliersForEditing((int) $product->id));
+        $productSuppliersForEditing = $this->getProductSuppliersForEditingHandler->handle(
```

Comment:
I thought we said handlers were not supposed to use other handlers internally any more
I guess this one was made after we made that decision, it would deserve a little refacto I guess (in another PR)

Reply:
yeah Im sure we did replace those usages in commands, but not queries.
btw this query might need a huge refacto after the use case have changed :thinking: 
I wonder if it deserves separate pr?